packages/next-auth/src/providers/hubspot.ts

```diff
@@ -0,0 +1,104 @@
+import type { OAuthConfig, OAuthUserConfig } from "."
+
+interface HubSpotProfile extends Record<string, any> {
+
+  // TODO: figure out additional fields, for now using 
+  // https://legacydocs.hubspot.com/docs/methods/oauth2/get-access-token-information
+
+  user: string,
+  user_id: string,
+
+  hub_domain: string,
+  hub_id: string,
+}
+
+
+const HubSpotConfig = {
+  clientId: process.env.HUBSPOT_CLIENT_ID ?? "DEFAULT_CLIENT_ID",
```

Comment:
I find these "DEFAULT_" values don't help much. Please just remove them 🙏

Reply:
@ThangHuuVu I've outsourced them to the provider instantiation with the required fields. Updated docs for that as well.